### PR TITLE
Add Intl article

### PR DESCRIPTION
**Pull request by mathiasbynens**

### CODE REVIEW

File: src/blog/intl.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+The `Intl` implementation was not modeled after such an architecture, as a consequence of the historic split. Instead, all the built-in JavaScript objects as specified by the Internationalization spec (like `NumberFormat`, `DateTimeFormat`) were generic `JSObject`s that had to transition through several property additions for their internal slots.
+
+Another artifact of not having a specialized `JSObject`s was that type checking was now more complex. The type information was stored under a private symbol and type-checked on both the JS and C++ side using expensive property access, rather than just looking up its shape.
+
+### Modernizing the codebase
+
+With the current move away from writing self-hosted builtins in V8, it made sense to use this opportunity to modernize the ECMA402 implementation.
```
Comment (sam-github) [inline src/blog/intl.md]: what does "self-hosted" mean?

Comment (mathiasbynens) [reply src/blog/intl.md]: Implemented in JS as part of V8. @gsathya, WDYT? Should we link to https://en.wikipedia.org/wiki/Self-hosting or explain this in a few words?

Comment (gsathya) [reply src/blog/intl.md]: Linking to the wikipedia article sounds good to me.

Comment (mathiasbynens) [reply src/blog/intl.md]: @sam-github Would you like to submit a pull request for this? :)

**Pull request merged**